lib/components/dashboard/Actionbar.tsx

```diff
@@ -383,6 +371,20 @@ const Actionbar: FC<{ mode: ReviewMode }> = ({ mode }) => {
               sx={{ px: 1 }}
             >
               <Reviewers />
+              <Tooltip
+                title="Click to get a shareable link to your plan."
+                placement="left"
+                arrow
+              >
+                <Button
+                  onClick={onShareClick}
+                  variant="outlined"
+                  sx={{ width: '10rem', height: '2rem', mx: 1, my: 1 }}
+                  color="info"
+                >
+                  <div className="ml-1">Copy Plan Link</div>
```

Comment:
Let's make the wording more clear and the UI a bit better. Maybe look at some other sites for inspiration maybe. 

Maybe reword to "Share Plan Snapshot", as it creates a copy of a snapshot of the current plan to someone else's account rather than adding them as a reviewer (the other function), where they can see the most updated version of the plan on their reviewer dashboard screen.

Reply:
Also the linter is failing please check the github action on why the linter is failing.